lib/bats-core/tracing.bash

```diff
@@ -77,7 +77,7 @@ bats_print_failed_command() {
 	bats_frame_lineno "$frame" 'lineno'
 	bats_extract_line "$filename" "$lineno" 'failed_line'
 	bats_strip_string "$failed_line" 'failed_command'
-	printf '%s' "#   \`${failed_command}' "
+	printf '%s' "#   '${failed_command}' "
```

Comment:
Again, I have no problem with \`. However, my OCD kicks in when I see that this is not `printf '#  "'s'" "$failed_command"`.